pkg/controller/projectclaim/projectclaim_controller.go

```diff
@@ -83,9 +80,25 @@ func (r *ReconcileProjectClaim) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	adapter := NewCustomResourceAdapter(instance, reqLogger, r.client)
+	result, err := r.ReconcileHandler(adapter)
+	if err != nil {
+		message := err.Error()
+		reason := "ReconcileFailed"
+		// Update the ProjectClaimConditionCRD
+		adapter.SetProjectClaimCondition(corev1.ConditionTrue, reason, message)
```

Comment:
We need to update the status of the ProjectClaim after updating the conditions so that we write the conditions to the CR.

Reply:
the status update is now being made in `SetProjectClaimCondition()`